code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -516,6 +522,35 @@
 		"is stabbing themselves with their mandibles!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/toggle_biolum,
+		)
+
+/mob/living/carbon/human/proc/toggle_biolum()
```

Comment:
It might be better if this were an action button; Look at the NODROP implant, or the IPC power port for examples of this